code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -34,6 +34,16 @@
 		GLOB.ark_of_the_clockwork_justiciar = src
 	START_PROCESSING(SSprocessing, src)
 
+/obj/structure/destructible/clockwork/massive/celestial_gateway/proc/final_countdown(ark_time)
+	if(!ark_time)
+		ark_time = 30 //minutes
+	initial_activation_delay = ark_time * 60
+	seconds_until_activation = ark_time * 60 //60 seconds in a minute * number of minutes
```

Comment:
can you please make this vars, and ark_time, actual decisecond values?

Reply:
In order to track the countdown to the second, the Ark counts down per-second, like it also did before the rework. Making them deciseconds would have to remove a chunk of functionality from the Ark and would require a refactor.

Reply:
world.time is deciseconds, so it's not that hard to change and keep the functionality.

Reply:
How would I implement it? Change everything the Ark uses to use deciseconds and world.time? `process()` already ticks once per second.

Reply:
🔔